core/help.py

```diff
@@ -172,9 +177,7 @@ async def send_cog_help(self, cog: commands.Cog) -> None:
             cog.get_commands(), sort=self.sort_commands
         )
         if filtered:
-            self.paginator.add_line(
-                f"{tr('help', 'module')} **__{cog.qualified_name}__**"
-            )
+            self.paginator.add_line(f"{tr('Module')} **__{cog.qualified_name}__**")
```

Comment:
You've missed one `tr()` call here.

Reply:
resolved in b4e6d7d0a064707ca1c52d507e9d138a2309a43e